package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "supercluster_data_portal_api",
   "version": "0.0.1",
-  "description": "",
+  "description": "Data Access Control API",
   "main": "src/index.ts",
   "scripts": {
     "build": "ttsc",
-    "start": "TS_NODE_BASEURL=./dist node -r tsconfig-paths/register ./dist/index.js",
     "dev": "ts-node ./src/index.ts",
-    "dev:nodemon": "nodemon -w src -e ts,json -x ts-node ./src/index.ts",
+    "dev:nodemon": "nodemon -e js,json,ts -x ts-node ./src/index.ts",
+    "start": "TS_NODE_BASEURL=./dist node -r tsconfig-paths/register ./dist/index.js",
+    "swagger": "swagger-jsdoc -d ./dist/swagger.js -o ./dist/swagger/documentation.json",
```

Comment:
add `swagger` npm script, and improve `nodemon`'s to restart the API when working on configs and/or with linked  dependencies (such as Arranger 😉)